http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeConfigTable.scala

```diff
@@ -45,17 +47,23 @@ class RuntimeConfigTable(tag: Tag) extends Table[RuntimeConfigRecord](tag, "RUNT
              workerDiskSize,
              numberOfWorkerLocalSSDs,
              numberOfPreemptibleWorkers,
-             dataprocProperties),
+             dataprocProperties,
+             persistentDiskId),
             dateAccessed) =>
         val r = cloudService match {
           case CloudService.GCE =>
-            RuntimeConfig.GceConfig(machineType, diskSize)
-
+            diskSize match {
+              case Some(size) => RuntimeConfig.GceConfig(machineType, size)
```

Comment:
Should this also be a `GceWithPdConfig`?

Reply:
no. only `RuntimeConfig.GceConfig` will have value for diskSize....`RuntimeConfig.GceWithPdConfig` will have a reference to a record in PD table, which has size etc